### PR TITLE
164365357 Add comment CRUD

### PR DESCRIPTION
**Pull request by mwinel**
**What does this PR do?**

This PR adds the **add-comment** API feature. An authenticated user can;
- add a comment or comments to a question
- read all comments to a particular question
- comment owners can update a comment
- comment owners can delete a comment

**Related Stories**
[#164365357](https://www.pivotaltracker.com/story/show/164365357)

@bisonlou, @kbjude, @mwinel @AtamaZack @PatrickMugayaJoel @kalsmic

### CODE REVIEW

File: question/views.py [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
 
+<<<<<<< HEAD
+from question.models import Question, Comment
+from question.serializers import QuestionSerializer, CommentSerializer
+from question.permissions import IsOwnerOrReadOnly
+=======
 from meetup.models import Meeting
 from question.models import Question, Vote
 from question.serializers import QuestionSerializer, VoteSerializer
+>>>>>>> 54e3da93010e1083a4b17abf2b76bcfc556c02a8
```
Comment (kalsmic) [inline question/views.py]: Please remove the merge conflict markers

**Review by kalsmic — CHANGES_REQUESTED** [review]

File: question/views.py (hunk 1 above)
Comment (kalsmic) [reply question/views.py]: The conflict markers are also part of the reason why travis is failing

Comment (mwinel) [reply question/views.py]: Sure. Just removed them.

**Comment by kalsmic** [comment]
@mwinel , I also noticed from the travis logs that the vote model is imported. Please import it 
https://github.com/kbjude/questioner/blob/ce9b54c3c42896087c766ede016d22d927189f44/question/serializers.py#L3
https://github.com/kbjude/questioner/blob/ce9b54c3c42896087c766ede016d22d927189f44/question/serializers.py#L39-L40

File: question/views.py (hunk 1 above)
Comment (AtamaZack) [reply question/views.py]: Hello @mwinel, the branch you plan to merge with is current stable (develop), I would recommend doing an upstream pull from it then add new code to it, else with the current Travis report, a merge would render develop unstable.

**Pull request closed without merging**